git_machete/client.py

```diff
@@ -2065,6 +2032,10 @@ def retarget_github_pr(self, head: LocalBranchShortName) -> None:
             print(fmt(f'The base branch of PR #{pr.number} is already `{new_base}`'))
 
     def __derive_remote_and_github_org_and_repo(self) -> RemoteAndOrganizationAndRepository:
+        remote_and_organization_and_repository_for_custom_url = self.__get_remote_and_organization_and_repository_name_for_custom_url()
```

Comment:
Pls provide some simple test that would actually retrieve the remote/org/repo data from config keys and NOT from `git remote` data